apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerTransactionCreationOptions.cs

```diff
@@ -105,7 +107,15 @@ public sealed class SpannerTransactionCreationOptions
     /// </remarks>
     public bool ExcludeFromChangeStreams { get; }
 
-    private SpannerTransactionCreationOptions(TimestampBound timestampBound, TransactionId transactionId, bool isDetached, bool isSingleUse, bool isPartitionedDml, bool excludeFromChangeStreams)
+
+    /// <summary>
```

Comment:
Update this documentation to explain what this field actually is, not how we are using it. Follow some of the other properties as an example.